dart/lib/src/sentry_baggage.dart

```diff
@@ -87,6 +90,27 @@ class SentryBaggage {
     return SentryBaggage(keyValues, logger: logger);
   }
 
+  @internal
+  setValuesFromScope(Scope scope, SentryOptions options) {
```

Comment:
On Java we are setting `transaction` and `sampleRate` to null. Is this also needed here?